Description: accept newer branch of crate env_logger
Author: Jonas Smedegaard <dr@jones.dk>
Bug-Debian: https://bugs.debian.org/1084849
Last-Update: 2024-10-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,7 +36,7 @@
 clap = { version = "4.4.18", features = ["derive", "env", "wrap_help"] }
 directories = "5.0.1"
 encoding_rs = "0.8"
-env_logger = "0.10"
+env_logger = ">= 0.10, <= 0.11"
 filetime = "0.2"
 flate2 = { version = "1.0", optional = true, default-features = false, features = [
   "rust_backend",
